<launch>

<!-- ARGUMENTS -->


<!-- PARAMETERS -->

<!--Loads parameters stores in YAML into the parameter server-->



<!-- Load URDF file to parameter server -->
  <arg name="leg_lenght" value="2.0" />

 <param name="robot_description"
  command="xacro --inorder '$(find robostilt_description)/urdf/robostilt.urdf.xacro' leg_lenght:=$(arg leg_lenght) " />


  <!-- read value of arg -->
  <param name="ReadFromArgumnet" value="$(arg leg_lenght)"/>

<!-- Launch files -->

<!-- Launch TF publishers -->


<!-- NODES -->

  <!-- Joint state publisher reads URDF and publishes all the non-fixed joints as JointState messages to sensor/JointState-->
  <!--<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" args="_use_gui:=false publish_default_positions:=true" /> -->

  <!-- Robot state publisher updates reads join state message and updates the TF tree-->
  <!-- <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />  -->



</launch>